app/timetables/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from app.timetables.models import Weekday
```

Comment:
@andela-ooshodi as regards `from __future__ import absolute_import` we are using python3 so this will not be necessary.